src/utils.jl

```diff
@@ -129,26 +135,23 @@ and FOE is to be expected. The equations to compute ̂M are from Appendix B of t
 Return true if the criterion is fulfilled, and it is ok to use GCDE.
 Return false if the criterion is not fulfilled, and FOE should be used instead.
 """
-function gcde_check(M::AbstractEquilibrium, gcp::GCParticle, path::OrbitPath; threshold=0.073, verbose=false)
+function _gcde_check(M::AbstractEquilibrium, KE::AbstractFloat, m::AbstractFloat, q::Int, r::AbstractVector{T}, phi::AbstractVector{T}, z::AbstractVector{T}, pitch::AbstractVector{T}; threshold=0.073, verbose=false) where {T<:AbstractFloat}
```

Comment:
Nice. However, there is now no function with the form gcde_check(M::AbstractEquilbrium, gcp::GCParticle, path::OrbitPath; ...) right? Then line 531 (519 in old version) of orbit.jl will fail?

Reply:
I found it three seconds later. I am blind. Ignore this review thread. It's all good.